collagraph/renderers/pyside/objects/standarditem.py

```diff
@@ -0,0 +1,47 @@
+from .qobject import set_attribute as qobject_set_attribute
+
+
+def insert(self, el, anchor=None):
+    if index := getattr(el, "model_index", None):
```

Comment:
0 is falsy, but seems like a valid index to me?

Reply:
`model_index` is a custom attribute that should be a tuple (row, column). A tuple with (0, 0) still evaluates to true, this should work. But I guess checking for None would be more explicit.

Reply:
I've updated this so that the value of `model_index` is always unpacked to `row, column` so that people will find out quickly if they supply something that can't be unpacked in such a way.
Also changed the if statement to `hasattr` instead, without a walrus.